src/RSSFeed.php

```diff
@@ -486,21 +486,30 @@ public function prepareInputForAdd($input)
         }
         $input['users_id'] = $current_user_id;
 
-        if ($feed = self::getRSSFeed($input['url'])) {
-            $input['have_error'] = 0;
-            $input['name']       = $feed->get_title();
-            if (empty($input['comment'])) {
-                $input['comment'] = $feed->get_description();
+        // We may want to disable the name computation when working with fake
+        // feeds in our unit tests
+        $compute_name = $input['_do_not_compute_name'] ?? false;
+        if (!$compute_name) {
+            if ($feed = self::getRSSFeed($input['url'])) {
+                $input['have_error'] = 0;
+                $input['name']       = $feed->get_title();
+                if (empty($input['comment'])) {
+                    $input['comment'] = $feed->get_description();
+                }
+            } else {
+                $input['have_error'] = 1;
+                $input['name']       = '';
             }
-        } else {
-            $input['have_error'] = 1;
-            $input['name']       = '';
         }
         $input["name"] = trim($input["name"]);
 
         if (empty($input["name"])) {
             $input["name"] = __('Without title');
         }
+
+        // Owner cannot be changed
+        unset($input['users_id']);
+
```

Comment:
This value is forced on line 487.
```suggestion
```

Reply:
Indeed, it comes from a bad rebase I think.